src/styles/sections/footer.scss

```
@@ -1,5 +1,13 @@
 .footer {
-  background: #202D33;
+  background: #202d33;
   text-align: center;
   padding: 30px;
-}
\ No newline at end of file
+}
+
+.cookies-note {
+  color: white;
```

```suggestion
  color: #ffffff;
```

Navrhoval bych definovat barvy v HEX číslech ať je to konzistentní. Vídím, že tam máme několik variant bíle barvy -`#FFFFFF`, `#fff` a `#ffffff`, co to třeba sjednotit a dát jednoho formátu? :)

---

Pardon, to mi uniklo. Běžně tu slovní variantu necommituju. Mám teda vytvořit nový pull request s opravou všech stylů? Formát #FFFFFF? Pro všechy barvy?

---

Ja bych to asi klidně udělal v tomhle PR, možná @HormCodes má jiný názor?

---

Za mne separé commit v tomto PR je OK.

---

Mimochodem good point. Můžeme to rovnou přidat do konvencí. 👍

---

LGTM, merguji a díky @poooow 👍